src/media/css/detail/header.styl

```diff
@@ -11,6 +11,9 @@
     .mkt-app-button {
         margin: 0 auto;
     }
+    .app-notices {
+        border: 0 !important;
```

Comment:
Let's not use `!important`, even if that means having to reshuffle CSS elsewhere.